packages/website/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "polaris-shopify-com",
```

Comment:
**thought:** `packages/website` might be too generic as there will likely be more websites hosted here. Wonder if we should be more intentional with the naming of the Polaris doc site. e.g. `packages/shopify`, [packages/docs](https://github.com/mui/material-ui/tree/master/docs), `packages/style-guide`, etc.

Reply:
I have a strong opinion that putting everything in `packages` is an extra click for everyone. Would prefer if we didn't use the packages approach. Probably going to get hated for this. Right now we have:

- Tokens
- Components
- Icons
- polaris.shopify.com
- Polaris for Figma
- Polaris for VSCode

I don't think the packages layer is needed for the next year.

Reply:
I think we could do both. I personally prefer and expect the convention of using a `packages/*` directory. However, I've mostly observed that this is only used for code that is published as an npm package. Thus, other workspaces could still live at the root-level. [Turborepo splits this up](https://github.com/vercel/turborepo/blob/main/pnpm-workspace.yaml) well by having published packages in the `packages/*` folder, but then other non-package code at the root (`docs`, `cli`, and `benchmark`).

This approach has the advantages of keeping published assets contained and organized while keeping others more discoverable. The heuristic could be defined–if it goes in a registry, it belongs in `packages`. Thoughts? @alex-page @aaronccasanova